src/components/Video/VideoHelp.vue

```diff
@@ -16,55 +16,47 @@
         <p class="col-xs-6">Move between recordings for device</p>
       </b-col>
       <b-col cols="5">
-        <font-awesome-layers class="fa-fw">
-          <font-awesome-icon 
```

Comment:
Add class fa-fw to the b-col to fix the colours (currently showing with opacity = 1 instead of 0.6)